host/xtest/xtest_main.c

```diff
@@ -156,21 +156,21 @@ int main(int argc, char *argv[])
 	init_ossl();
 
 	if (argc > 1 && !strcmp(argv[1], "--sha-perf"))
-		return hash_perf_runner_cmd_parser(argc-1, &argv[1]);
+		return hash_perf_runner_cmd_parser(argc, argv);
```

Comment:
How about as first thing save `argv[0]` in a global variable? It should save us from having to do the same thing again and again.

Reply:
ok